docs/en/resources/sources/cloud-sql-admin.md

```diff
@@ -37,6 +36,7 @@ sources:
 ## Reference
 
 | **field**      | **type** | **required** | **description**                                                                                                                                |
-|----------------|:--------:|:------------:|------------------------------------------------------------------------------------------------------------------------------------------------|
+| -------------- | :------: | :----------: | ---------------------------------------------------------------------------------------------------------------------------------------------- |
 | kind           |  string  |     true     | Must be "cloud-sql-admin".                                                                                                                     |
-| useClientOAuth | boolean  |    false     | If true, the source will use client-side OAuth for authorization. Otherwise, it will use Application Default Credentials. Defaults to `false`. |
+| defaultProject |  string  |     false    | For prebuilt tools, this is the environment variable containing the Google Cloud project ID.                                                   |
```

Comment:
Please add any prebuilt info to the reference page: https://googleapis.github.io/genai-toolbox/reference/prebuilt-tools/. These doc pages should just reference the field ie "The Google Cloud project ID to use for Cloud SQL infrastructure tools."